#! /bin/sh /usr/share/dpatch/dpatch-run
## gpsfake-logfile-howto.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: Fixing a bug which was asking for a description on how to create
## DP: logfiles suitable for gpsfake.

@DPATCH@
diff -urNad gpsd~/gpsfake.xml gpsd/gpsfake.xml
--- gpsd~/gpsfake.xml	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/gpsfake.xml	2007-10-19 06:58:49.000000000 +0200
@@ -128,6 +128,10 @@
 torture-test either <application>gpsd</application> itself or any
 <application>gpsd</application>-aware client application.</para>
 
+<para>Logfiles for the use with <application>gpsfake</application> can be
+retrieved using <application>gpspipe</application> and
+<application>sirfmon</application> from the gpsd distribution, or any other
+application which is able to create a compatible output.</para>
 </refsect1>
 <refsect1 id='see_also'><title>SEE ALSO</title>
 <para>
@@ -136,8 +140,9 @@
 <citerefentry><refentrytitle>libgps</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
 <citerefentry><refentrytitle>libgpsd</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
 <citerefentry><refentrytitle>gpsctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-<citerefentry><refentrytitle>gpscat</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
-<citerefentry><refentrytitle>gpsprof</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+<citerefentry><refentrytitle>gpspipe</refentrytitle><manvolnum>1</manvolnum></citerefentry>,
+<citerefentry><refentrytitle>gpsprof</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+<citerefentry><refentrytitle>sirfmon</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
 </para>
 </refsect1>
 
